develop.yml

```diff
@@ -21,9 +21,11 @@ services:
       - /config/run_idx.sh
     depends_on:
       - mongo
+    tty: true
+    stdin_open: true
   back:
     build:
-      context: "https://github.com/camicroscope/caracal.git#develop"
+      context: "../Caracal"
```

Comment:
Remember to change this back when you're ready for review.

Reply:
Yes, I'll change this for a review. However, note that the user dashboard feature done by me won't be working until [#333](https://github.com/camicroscope/caMicroscope/pull/333) and [#21](https://github.com/camicroscope/Caracal/pull/21) gets merged. 

I think it will be better if we create a new branch for mongo 4.0 and test the entire application on it? I'll also submit this and the other two pull requests on that branch. Then we can merge the new branch into develop